l10n_ch_base_bank/models/bank.py

```diff
@@ -235,34 +235,12 @@ def _check_postal_num(self):
                 )
         return True
 
-    @api.multi
-    def _get_acc_name(self):
-        """ Return an account name for a bank account
-        to use with a ccp for ISR.
-        This method makes sure to generate a unique name
-        """
-
-        part_name = self.partner_id.name
-        if not part_name and self.env.context.get('default_partner_id'):
-            partner_id = self.env.context.get('default_partner_id')
-            part_name = self.env['res.partner'].browse(partner_id)[0].name
-
-        if part_name:
-            acc_name = _("Bank/CCP {}").format(part_name)
-        else:
-            acc_name = _("Bank/CCP Undefined")
-
-        exist_count = self.env['res.partner.bank'].search_count(
-            [('acc_number', '=like', acc_name)])
-        if exist_count:
-            name_exist = exist_count
-            while name_exist:
-                new_name = acc_name + " ({})".format(exist_count)
-                name_exist = self.env['res.partner.bank'].search_count(
-                    [('acc_number', '=', new_name)])
-                exist_count += 1
-            acc_name = new_name
-        return acc_name
+    @api.model
+    def create(self, vals):
+        # One of the fields is mandatory
+        if not vals.get('acc_number') and vals.get('ccp'):
+            vals['acc_number'] = vals['ccp']
```

Comment:
This will probably not work with 2 partners having the same ccp.

As we still have the constraint:

https://github.com/odoo/odoo/blob/11.0/odoo/addons/base/res/res_bank.py#L74